### DISCUSSION TITLE
Please create a no reboot option for command line

### DISCUSSION THREAD

**Discussion by dric322** [ionuttbara/windows-defender-remover#84] [Ideas]
I want to include this as part of an unattended Windows 11 install (using option "n") but it can only go as the last item being run because of the forced reboot after 5 seconds.

It would be good to have an option like "/N /norestart" on the command line. Hint hint :p

**Comment by ionuttbara** [comment]
**No!**
The modification (disable/enable) it takes effect after reboot and its needed that.
